test/end-to-end/reports/balance/balance.spec.js

```diff
@@ -0,0 +1,80 @@
+/* global browser, element, by */
+
+const chai = require('chai');
+const expect = chai.expect;
+
+const components = require('../../shared/components');
+const FU = require('../../shared/FormUtils');
+const helpers = require('../../shared/helpers');
+const moment = require('moment');
+helpers.configure(chai);
+
+describe('Balance Report Generation', () => {
+  'use strict';
+
+  before(() => helpers.navigate('#/reports/balance'));
+
+  const annualReport = {
+    title: 'Balance Annuelle 2016',
+    dateFrom: '01/01/2016',
+    dateTo: '31/12/2016',
+    dateOption: 0,
+    classe: '*'
+  };
+
+  const reportDate = new Date();
+  const humanDate = moment(reportDate).format('DD MMMM YYYY');
+
+  const report = {
+    title: 'Balance jusqu\'a la date ' + humanDate,
+    date: reportDate,
+    dateOption: 1,
+    classe: '*'
+  };
+
+  it('GET /reports/finance/balance return an annual balance report to the client', () => {
```

Comment:
Using 'GET' in an end-to-end test is a bit strange.  It's the browser that sends the GET request!